src/voicecall.c

```diff
@@ -117,8 +117,8 @@ struct emulator_status {
 };
 
 static const char *default_en_list[] = { "911", "112", NULL };
-static const char *default_en_list_no_sim[] = { "119", "118", "999", "110",
-						"08", "000", NULL };
+static const char *default_en_list_no_sim[] = { "000", "08", "110", "118",
+	"119", "120", "122", "999",  NULL };
```

Comment:
Please append 120 and 122 to the list and do not change the order, to ease comparison with older version.